File: ape.java
class Ape{
	private int hands;
	private int legs;

	public Ape(){

	}
	public Ape(int h,int l){
		hands =h;
		legs = l;
	}
	public int getLegs(){
		return legs;
	}

	public int getHands(){
		return hands;
	}

	public void setHands(int h) {
		if(h>=0 && h<=2)
		hands = h;
	}
 
	public void setLegs(int l) {
		if(l>=0 && l<=2)
		legs = l;
	}
}
class Human extends Ape{
	private String name;

	public Human(){
		super();
	}
	public Human(String str,int h,int l)
	{
		super(h,l);
		name = str;
	}

	public String getName(){
		return name;
	}

	public void setName(String str){
		name = str;
	}

	public int  getHands()
	{
		return super.getHands() + 1; 
	}
	public int getHands()
	{
		return super.getHands();
	}
}

public class ape{

	public static void main(String[] args){
		Human a = new Human("human" , 2,2);
		// System.out.println( a.getName()+ "has "  + a.getHands() + " hands " + a.getLegs() + " legs");
		System.out.println("hands " + a.getHands());
	}
}	

File: ape.cpp
#include <iostream>
using namespace std;
 
class ape{
	private:
	int hands;
	int legs;
 
	public:
 
	/*ape() {
		hands = 2;
		legs = 2;
	}
 */
	int getHands() {
		return hands;
	}
 
	int getLegs() {
		return legs;
	}

	void setHands(int h) {
		if(h>=0 && h<=2)
		hands = h;
	}
 
	void setLegs(int l) {
		if(l>=0 && l<=2)
		legs = l;
	}
 
	
};
 
 
 
int main() {
	ape a;
	cout<<a.getHands()<<"\n";

	a.setHands(1);
	cout<<a.getHands()<<"\n";
	a.setHands(3);
	cout<<a.getHands()<<"\n";

	return 0;
}